src/widgets/_index.scss

```diff
@@ -18,28 +18,35 @@ messagedialog,
 @define-color success_color @LIME_700;
 @define-color warning_color @BANANA_900;
 
+@define-color selected_bg_color #{"alpha(@accent_color_100, 0.3)"};
+@define-color selected_fg_color #{"@accent_color_900"};
+
 // Things Dazzle expects
 @define-color borders #{"" + $toplevel-border-color};
 @define-color content_view_bg #{""+ bg_color(2)};
 @define-color theme_fg_color #{"" + $fg-color};
-@define-color theme_selected_fg_color #{"" + $fg-color};
+@define-color theme_selected_bg_color #{"@selected_bg_color"};
+@define-color theme_selected_fg_color #{"@selected_fg_color"};
 @define-color wm_shadow #{rgba(black, 0.2)};
 
-@define-color selected_bg_color #{'alpha(@accent_color_100, 0.3)'};
-@define-color selected_fg_color #{'@accent_color_900'};
+selection {
+    @extend %selected_item;
+}
+
+.gtkstyle-fallback {
+    @extend %selected_item;
+}
 
 @if $color-scheme == "dark" {
     @define-color error_color @STRAWBERRY_300;
     @define-color success_color @LIME_300;
     @define-color warning_color @BANANA_100;
 
-    @define-color selected_bg_color #{'alpha(@accent_color_700, 0.3)'};
-    @define-color selected_fg_color #{'@accent_color_100'};
-}
+    @define-color selected_bg_color #{"alpha(@accent_color_700, 0.3)"};
+    @define-color selected_fg_color #{"@accent_color_100"};
```

Comment:
Can you undo this change to quotations please

Reply:
Isn't `"` more correct than `'` in Sass for this thing? The official docs say one uses `"` so I'm confused.

Reply:
https://sass-lang.com/documentation/interpolation#quoted-strings

Relevant docs